src/style.css

```diff
@@ -0,0 +1,77 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.search__form {
+  background-color: #fff;
+  position: relative;
+  margin-top: 20px ;
+  border: 1px solid  white;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+
+}
+
+.search__form--txt {
+  display: flex;
+  width: 100%;
+  height: 70px;
+  border: none;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+  font-family: "Avenir", sans-serif;
+}
+
+.icon {
+  display: block;
+  position: absolute;
+  left: 26px;
+  top: 25px;
```

Comment:
It's a bad way to center an icon vertically: https://www.w3schools.com/howto/howto_css_center-vertical.asp there is a couple of ways to do it